templates/search.html

```diff
@@ -3,7 +3,7 @@
 <!-- wp:group {"tagName":"main","style":{"spacing":{"margin":{"top":"var:preset|spacing|60"}}},"layout":{"type":"constrained"}} -->
 <main class="wp-block-group" style="margin-top:var(--wp--preset--spacing--60)">
 	<!-- wp:query-title {"type":"search"} /-->
-	<!-- wp:search {"label":"Search","showLabel":false,"buttonText":"Search","style":{"border":{"radius":"50px"}},"fontSize":"medium"} /-->
+	<!-- wp:pattern {"slug":"twentytwentyfive/search"} /-->
```

Comment:
We'll need to update this one once we change the filename.

```suggestion
	<!-- wp:pattern {"slug":"twentytwentyfive/hidden-search"} /-->
```

Reply:
@juanfra this change has been made. I didn't know about the hidden pattern syntax. Thanks for explaining :)